tests/integration/standard/test_concurrent.py

```diff
@@ -47,6 +50,24 @@ def setUpClass(cls):
     def tearDownClass(cls):
         cls.cluster.shutdown()
 
+    def execute_concurrent_helper(self, session, query):
+        while True:
+            try:
+                return execute_concurrent(session, query)
+            except (ReadTimeout, WriteTimeout, OperationTimedOut, ReadFailure, WriteFailure):
+                ex_type, ex, tb = sys.exc_info()
+                log.warn("{0}: {1} Backtrace: {2}".format(ex_type.__name__, ex, traceback.extract_tb(tb)))
+                del tb
+
+    def execute_concurrent_args_helper(self, session, query, params):
+        while True:
+            try:
+                return execute_concurrent_with_args(session, query, params)
+            except (ReadTimeout, WriteTimeout, OperationTimedOut, ReadFailure, WriteFailure):
+                ex_type, ex, tb = sys.exc_info()
+                log.warn("{0}: {1} Backtrace: {2}".format(ex_type.__name__, ex, traceback.extract_tb(tb)))
+                del tb
+
```

Comment:
The title of the method is `execute_concurrent_args_helper`. Not sure how else I can specify that it's a helper.